webapp/src/js/services/db-sync.js

```diff
@@ -167,6 +167,11 @@ angular
         updateListeners.push(listener);
       },
 
+      /**
+       * Boolean representing if sync is curently in progress
+       */
+      isSyncInProgress: () => inProgressSync,
```

Comment:
The doc says "Boolean" but it's actually a "Promise" - a boolean is what I'd expect so maybe change the implementation to `!!inProgressSync`?